src/main/java/marquez/client/utils/JsonUtils.java

```diff
@@ -19,21 +19,15 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
-import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import io.dropwizard.jackson.Jackson;
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import lombok.NonNull;
 
 public final class JsonUtils {
   private JsonUtils() {}
 
-  private static final ObjectMapper MAPPER =
-      new ObjectMapper()
-          .registerModule(new Jdk8Module())
-          .registerModule(new JavaTimeModule())
-          .registerModule(new ParameterNamesModule());
+  public static final ObjectMapper MAPPER = Jackson.newObjectMapper();
```

Comment:
_Minor:_ We'll want to define a factory method for the object mapper so we can reuse the instance in tests, for example:

```java
JsonUtils.newObjectMapper()
```